setup.py

```diff
@@ -0,0 +1,41 @@
+from setuptools import setup
+
+CLASSIFIERS = [
+    "Development Status :: 4 - Beta",
+    "Natural Language :: English",
+    "License :: OSI Approved :: GNU General Public License v2 (GPLv2)",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.3",
+    "Programming Language :: Python :: 3.4",
+    "Programming Language :: Python :: 3.5",
+    "Programming Language :: Python :: 3.6",
+    "Programming Language :: Python :: Implementation :: PyPy",
+    "Topic :: Scientific/Engineering :: Bio-Informatics",
+]
+
+setup(
+    name='sctools',
+    version='0.1.0',
+    description='Utilities for large-scale distributed single cell data processing',
+    url='https://github.com/humancellatlas/scsequtil.git',
+    author='Ambrose J. Carr',
+    author_email='mail@ambrosejcarr.com',
+    package_dir={'': 'src'},
+    packages=['sctools', 'sctools/test'],
+    test_suite='setup.sctools_test_suite',  # runs tests twice for who knows what reason
+    install_requires=[
+        'pysam',
+        'numpy',
+        'google-cloud',
```

Comment:
This brings in a huge web of dependencies. We were doing the same in secondary-analysis but changed it to install a more targeted set of google libraries. (We ran into an issue with one of the many packages that google-cloud installs, which it turned out we didn't actually need.)

Reply:
Ah, very cool. Do you think it would be adequate to pull the same set of libraries you're using in the secondary-analysis repo?

Reply:
Not sure, depends which actual google modules you're using. Are you using it just for reading from buckets? In that case you probably just need google-cloud-storage.

Reply:
I'll verify, but I think that's all.